packages/app/src/reducers/visualization.js

```diff
@@ -19,3 +19,5 @@ export default (state = DEFAULT_VISUALIZATION, action) => {
 // Selectors
 
 export const sGetVisualization = state => state.visualization;
+export const sGetInterpretations = state =>
+    state.visualization ? state.visualization.interpretations : [];
```

Comment:
here you can use `sGetVisualization(state).interpretations : [];`

Reply:
Hmm... default for visualization is {null}, and I think it was changed from `{}`.
I would leave that as it is.